libs/blocks/global-navigation/global-navigation.js

```diff
@@ -637,9 +637,12 @@ class Gnav {
     const state = getMetadata('app-prompt')?.toLowerCase();
     const entName = getMetadata('app-prompt-entitlement')?.toLowerCase();
     const promptPath = getMetadata('app-prompt-path')?.toLowerCase();
+    const hasMobileUA = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini|Touch/i.test(navigator.userAgent);
```

Comment:
nit: wouldn't isMobile be clearer?

Reply:
We have checks like `isDesktop` that check viewport size, which is what we check by default. I'd like to keep this name to differentiate it from the viewport size checks.